mysql-schema/complaint-table.txt

```diff
@@ -0,0 +1,14 @@
+CREATE DATABASE `hclecomm` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci */ /*!80016 DEFAULT ENCRYPTION='N' */;
```

Comment:
@rakeshkumar-verma : details for my-sql folder needs to go to readme.md

Reply:
@rishabh-behl  updated readme.md

Reply:
@rakeshkumar-verma : redme.md not part of thi PR . Pl add.